emacs/Readme.md

```diff
@@ -0,0 +1,2 @@
+## Acknowledgements
+This emacs major mode has been developed by [Reto Zimmermann](https://guest.iis.ee.ethz.ch/~zimmi/) (maintainer) and [Rod Whitby](http://www.rwhitby.net/) and published on their [website](https://guest.iis.ee.ethz.ch/~zimmi/emacs/vhdl-mode.html).
```

Comment:
Do we run into some license issue by directly including emacs vhdl mode?
We are randomly using MIT license I think, how compatible is this?

Reply:
They don't provide any license on their website nor in the dowloaded archive. Therefore, I assume it is fine with this acknowledgement.